sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
Ah, I was thinking that there might be more than one client (go vs ruby as an example) - so having the go version in the root felt odd to me.  But perhaps I was assuming too much.